Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -182,13 +189,17 @@ u16 VideoBackendBase::Video_GetBoundingBox(int index)
 void VideoBackendBase::PopulateList()
 {
   // OGL > D3D11 > Vulkan > SW > Null
+#ifdef HAS_OPENGL
   g_available_video_backends.push_back(std::make_unique<OGL::VideoBackend>());
+#endif
 #ifdef _WIN32
   g_available_video_backends.push_back(std::make_unique<DX11::VideoBackend>());
   g_available_video_backends.push_back(std::make_unique<DX12::VideoBackend>());
 #endif
   g_available_video_backends.push_back(std::make_unique<Vulkan::VideoBackend>());
+#ifdef HAS_OPENGL
```

Comment:
Uh, why is the Software Renderer dependant on OpenGL? That sounds...odd?

Reply:
It uses OpenGL for scanout but not rendering.